data/mlox_base.txt

```diff
@@ -28127,82 +28127,65 @@ Mage Friendly Bloodmoon.esp
 
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
-;; @Magic Diversity 1.0 [Arkann]
+;; @Magic Diversity [Arkann]
 
 [Conflict]
  Use only one plugin
-  "Magic Diversity 1.0.ESP" - The standard mod which modifies spell sounds.
+  "Magic Diversity <ver>.ESP" - The standard mod which modifies spell sounds.

```

Comment:
When using the predicates like VER and DESC I've always put them in capitals.

Doesn't affect the parsing, it's just more apparent to the Mk. 1 Human Eyeball